x-pack/legacy/plugins/ml/public/components/navigation_menu/tabs.tsx

```diff
@@ -95,18 +91,21 @@ export const Tabs: FC<Props> = ({ tabId, disableLinks }) => {
 
   return (
     <EuiTabs>
-      {tabs.map((tab: Tab) => (
-        <EuiTab
-          className="mlNavigationMenu__tab"
-          onClick={() => onSelectedTabChanged(tab.id)}
-          isSelected={tab.id === selectedTabId}
-          disabled={tab.disabled}
-          key={`${tab.id}-key`}
-          data-test-subj={TAB_TEST_SUBJ_MAP[tab.id]}
-        >
-          {tab.name}
-        </EuiTab>
-      ))}
+      {tabs.map((tab: Tab) => {
+        const id = tab.id;
```

Comment:
nit: you're assigning `id`, but then use a mix of `id` and still `tab.id` later on.

Reply:
Updated in b48feda7ed3d0d8103c0c0fd861a4564ba51f876